Pipfile

```diff
@@ -6,6 +6,7 @@ name = "pypi"
 [packages]
 pytest = "*"
 boto3 = "*"
+pydocstyle = "*"
```

Comment:
This should be under `[dev-packages]`, no?

Reply:
Uhhh I'm not sure why that's changed strange

Reply:
git weirdness 🤷‍♀️ Probably a good idea to remove it though.